modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPreloader.java

```diff
@@ -331,6 +347,15 @@ private IgniteCheckedException stopError() {
             }
         }
 
+        if (! skippedPartitionsLackHistSupplier.isEmpty())
+            log.info("Unable to perform historical rebalance cause " +
+                "history supplier is not available [parts=" + S.compact(skippedPartitionsLackHistSupplier) +
+                ", topVer=" + topVer + ']');
+
+        if (! skippedPartitionsCleared.isEmpty())
+            log.info("Unable to perform historical rebalance cause partition " +
+                "is supposed to be cleared [part=" + S.compact(skippedPartitionsCleared) + ", topVer=" + topVer + ']');
```

Comment:
Same as above +
Message better be changed to Unable to perform historical rebalance because clearing is required for partitions [parts=... and logged only if historical supplier exists.